### PR TITLE
WMS-535 | Percy Improvements

### PR DESCRIPTION
**Pull request by PedroHLC**
#### :thinking: What?
@renanpvaz  already removed Chrome and Mobile screenshots in Percy. This PR completes this by grouping components variations on one page.

* I decided to keep tables separated as they are more complex and way larger.

#### :man_shrugging: Why?
Cause we were breaking the free quotes.

#### :pushpin: Jira Issue
[WMS-535](https://paacklogistics.atlassian.net/browse/WMS-535)

#### :no_good: Blocked by
Not blocked, based on master.

#### :clipboard: Pending
Nothing.

### :fire: Extra
Feel free to discuss the table decision.

### CODE REVIEW

File: showcase/src/Badges.elm [hunk 1]
```diff
@@ -39,14 +41,40 @@ oneBadge cfg constructorLight constructorDark variation =
...
+        |> List.map (\constructor -> Badge.renderElement cfg (constructor "987"))
+        |> Element.row [ Element.spacing 8 ]
+
+
 code : String -> String
 code variation =
-    prettifyElmCode """
-lightOne renderConfig =
-    Badge.""" ++ variation ++ """Light "123"
+    prettifyElmCode <|
```
Comment (PedroHLC) [inline showcase/src/Badges.elm:68]: I've also fixed this small issue where elm code wasn't highlighting :sweat_smile:  (It was I who put this bug there)

File: cypress/integration/visual-regression.spec.js [hunk 2]
```diff
@@ -44,138 +29,38 @@ describe('Percy Snapshots', () => {
...
-    it('Creates a snapshot for full-width text field', () => {
-      cy.visit('/#Basics/TextField/Full%20Width')
-      cy.percySnapshot('Full-width text field')
+      cy.visit('/#Basics/TextField/All')
+      cy.percySnapshot('All text fields')
     })
   })
 
-  describe('Loading spinners', () => {
+  /*describe('Loading spinners', () => {
```
Comment (PedroHLC) [inline cypress/integration/visual-regression.spec.js:58]: Spinners don't seem to sync with Percy, always triggering the difference and unnecessarily delaying PRs.
But if we keep them we may notice if one day they disappear.
What do you guys think?

Comment (renanpvaz) [reply cypress/integration/visual-regression.spec.js:58]: supposedly percy pauses the animations to take the screenshot, there must be some delay causing that to not always happen at the same frame

Comment (renanpvaz) [reply cypress/integration/visual-regression.spec.js:58]: removing it should be okay

**Review by renanpvaz — APPROVED** [review]

File: cypress/integration/visual-regression.spec.js [hunk 3]
```diff
@@ -44,138 +29,38 @@ describe('Percy Snapshots', () => {
...
-
-    it('Creates a snapshot for password text field', () => {
-      cy.visit('/#Basics/TextField/Password')
-      cy.percySnapshot('Password text field')
-    })
-
-    it('Creates a snapshot for full-width text field', () => {
-      cy.visit('/#Basics/TextField/Full%20Width')
-      cy.percySnapshot('Full-width text field')
+      cy.visit('/#Basics/TextField/All')
```
Comment (jouderianjr) [inline cypress/integration/visual-regression.spec.js:53]: Are you sure this story exists? 🤔

**Review by jouderianjr — CHANGES_REQUESTED** [review]

File: src/UI/Checkbox.elm [hunk 4]
```diff
@@ -24,11 +24,6 @@ module UI.Checkbox exposing
 @docs withLabelVisible
 
 
-# Size
```
Comment (PedroHLC) [inline src/UI/Checkbox.elm:27]: Small docs fix here.
Maybe I should add docs to CI?

Comment (jouderianjr) [reply src/UI/Checkbox.elm:27]: @PedroHLC  that's a really good point, I will create a card for that

**Review by jouderianjr — APPROVED** [review]

**Pull request merged**